airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/SettingsView.tsx

```diff
@@ -15,11 +16,12 @@ interface SettingsViewProps {
 const SettingsView: React.FC<SettingsViewProps> = ({ connection }) => {
   const { mutateAsync: deleteConnection } = useDeleteConnection();
 
+  const [isAdvancedMode] = useAdvancedModeSetting();
   const onDelete = () => deleteConnection(connection);
 
   return (
     <div className={styles.container}>
-      <StateBlock connectionId={connection.connectionId} />
+      {isAdvancedMode && <StateBlock connectionId={connection.connectionId} />}
```

Comment:
Would it make sense to add a brief note here in the case that advanced mode is disabled, along the lines of `If you would like to view the state blob for your connection, enabled 'Advanced mode' in the Workspace settings`

I think this would be helpful to inform users that advanced mode exists, and it would instruct users who currently expect to see connection state here on how they can bring it back.

Reply:
Good suggestion--I've been a little bothered by the "user is surprised to see connection state UI has disappeared and doesn't know how to get it back" scenario, too.

Reply:
I haven't found a decent reference for the sort of unobtrusive info callout this should be; I'm inclined to ask for a design and put it off implementing it for a follow-up PR.

(left this comment in drafts for an hour by accident, at which point Lake had already posted a similar finding on Slack; nonetheless, putting it up here for others/posterity)

Reply:
The issue with merging this before we've added this callout is that we may end up with users in the confusing scenario we described in this comment thread.

You could always try adding something simple-looking here for now and then we can ask design to give us something, at which point we can update the look. Not sure how we usually handle situations like this though.

@timroes do you have any thoughts here?

Reply:
Discussed this earlier offline: We think it's best not to put the warning into the product for now, and rather leverage a tool like Intercom in our cloud to show users a banner that are looking for this, so we don't have "migration code" sitting around for too long (again).

Reply:
Sure, makes sense. Should we set up that intercom messaging before we deploy this change then?

Reply:
I just reached out to Natalie about that, seems like the right move to me. Doing it smoothly will take a little coordination (it would be pretty weird to see a popup about how "Connection State" has gone missing right next to "Connection State" after all), so I'll need a better handle on the deployment timing than I have now, but otherwise seemslike it might be pretty straightforward.